docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -64,13 +92,13 @@ To enable your application to use the media controller functionality:
 
 To update the metadata and playback information on the server side:
 
-1. Create the media controller server handle using the `mc_server_create()` function:
+1. Create the media controller server handle using the `mc_server_create()`:
 
    ```
    ret = mc_server_create(&g_server_h);
    ```
 
-2. Set the metadata or playback information to be updated using the corresponding `mc_server_set_XXX()` function, and then update the metadata or playback information using the corresponding `mc_server_update_XXX()` function.
+2. Set the metadata or playback information to be updated using the corresponding `mc_server_set_XXX()`, and then update the metadata or playback information using the corresponding `mc_server_update_XXX()`.
 
    For example, to update the playback state information, set the information to be updated using the `mc_server_set_playback_state()` function, and then update the information using the `mc_server_update_playback_info()` function:
```

Comment:
Remove 'function' word at both the places.

Reply:
OK, it will be changed.